hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -17,20 +17,22 @@
 from .reg import RegFile
 from .trace import Trace, TracePC
 from .wsr import WSRFile
+import sys
```

Comment:
This was for debugging prints, but I assume is not needed in the checked-in code.